app/views/Settings.js

```diff
@@ -73,18 +104,17 @@ export const SettingsScreen = ({ navigation }) => {
       console.log('something went wrong in lang change', e);
     }
   };
-
   return (
     <NavigationBarWrapper
-      title={t('label.settings_title')}
+      title={languages.t('label.settings_title')}
       onBackPress={backToMain}>
       <ScrollView>
         <Section>
           <Item
             label={
               isLogging
-                ? t('label.logging_active')
-                : t('label.logging_inactive')
+                ? languages.t('label.logging_active')
```

Comment:
For now it's probably ok to show "Location Inactive" for any of the "canTrack: false" states:

```js
const {canTrack, reason} = await LocationServices.checkStatus();

const isLogging = canTrack;
```

Here are the states btw:

- user toggled off inside the app
  ```js
  reason == Reason.USER_OFF
  ```
- permissions are not granted
  ```js
  reason == Reason.NOT_AUTHORIZED
  ```
- location services is turned off
  ```js
   reason == Reason.LOCATION_OFF
   ```

Eventually we would like to differentiate between these states.. but it's **ok to stick with `setIsLogging(canTrack)`** until the UX team gives guidance.

Feel free to reach out to them on slack if you want to incorporate more specific messaging here.